xbmc/test/TestUtils.cpp

```diff
@@ -7,6 +7,7 @@
  */
 
 #include "TestUtils.h"
+
```

Comment:
One more time: Remove this empty line. This is only a leftover from a previous iteration of the PR. You do not want to change this file.